import/CoordinateConversion.qml

```diff
@@ -44,22 +44,6 @@ Control {
      */
     property var geoView;
 
-    /*!
```

Comment:
just double-checking these changes are supposed to be in this PR?

Reply:
Sorry it was some minor cleanup when I was discovering some popupview properties I could apply.